src/components/NavigationBar.tsx

```diff
@@ -0,0 +1,41 @@
+import { AppBar, Toolbar, CssBaseline, Typography, ThemeProvider, createTheme, Link } from '@mui/material';
+import { SideBar } from './SideBar';
+import { Divider } from '@mui/material';
+
+const theme = createTheme({
+  palette: {
+    background: {
+      paper: '#fef8f8'
+    },
+    text: {
+      primary: '#3acff0'
+    },
+    divider: '#97948b'
+  }
+});
+
+function NavBar() {
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
We can probably remove this theme provider since `NavBar` is contained in the higher level Themeprovider

Reply:
Resolved